dvc/repo/__init__.py

```diff
@@ -25,11 +24,11 @@
 logger = logging.getLogger(__name__)
 
 
-def locked(f):
-    @wraps(f)
+def locked(func):
+    @wraps(func)
     def wrapper(repo, *args, **kwargs):
         with repo.lock, repo.state:
-            ret = f(repo, *args, **kwargs)
+            ret = func(repo, *args, **kwargs)
             # Our graph cache is no longer valid after we release the repo.lock
             repo._reset()
```

Comment:
@casperdcl These fixes are absolutely not related to the focus this PR, right? Please try not to do those in the same PR, it gets in the way of reviewing things that matter. 🙂

Reply:
will ignore DeepSource/condeclimate in future